json/src/main/kotlin/ktx/json/serializer.kt

```diff
@@ -0,0 +1,42 @@
+package ktx.json
+
+import com.badlogic.gdx.utils.Json
+import com.badlogic.gdx.utils.JsonValue
+
+/**
+ * Wrapping interface around [com.badlogic.gdx.utils.Json.Serializer].
+ * Provides null-safety for the methods.
+ */
+interface KtxSerializer<T> : Json.Serializer<T> {
```

Comment:
Not entirely sold on the names. I know there are examples of classes with `Ktx` prefix, but it's usually because otherwise it would lead to name duplications. In this case it's pretty easy to create descriptive names with no collisions.

* `Json.Serializer` -> `JsonSerializer`
* `Json.ReadOnlySerializer` -> `ReadOnlyJsonSerializer`

What do you think?

Reply:
Yes, I'm totally fine the names you suggested. I thought it was some kind of project convention for type names and decided to add `Ktx` prefix.